modules/ROOT/pages/save-builtin-dashboard-as-custom.adoc

```diff
@@ -0,0 +1,22 @@
+= Save a Built-in Dashboard as a Custom Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To simplify the process of creating a custom dashboard from scratch, you can save any of the built-in dashboards that are included in the Anypoint Monitoring tabs as a base template for a custom dashboard that you can then customize to suit your needs. This allows you to take advantage of the flexibility and features offered in custom dashboards, such as troubleshooting, setting up alerts, or comparing multiple resources within one dashboard. 
+
+Follow these steps to save a built-in dashboard as a custom dashboard. 
+
+. In Anypoint Monitoring, click *Built-in dashboards* in the menu on the left.
+. Click the tab for the built-in dashboard to save, for example, *Performance*. 
+. Click the *Save as* icon. 
+. In the *Save as* dialog, enter a *Name* for your custom dashboard, and click *Save*. +
+A green confirmation banner appears in the top of the screen. 
+. Click the *View in custom dashboards* link in the green confirmation banner to see the new custom dashboard in the list of custom dashboards. +
+The banner disappears after a moment. If you didn't click the link in the banner, you can click *Custom dashboards* in the menu on the left to go to the list of custom dashboards. +
+You can now configure your custom dashboard. 
+
```

Comment:
I think we should add something regarding this: You can't save a dashboard with a name that already exists.

Also we could add: After going to Custom dashboards, there you'll see the dashboard you have just saved with the name you entered

This is great!